src/Compilers/CSharp/Portable/Symbols/Source/CustomModifierUtils.cs

```diff
@@ -89,18 +89,21 @@ internal static TypeSymbol CopyTypeCustomModifiers(TypeSymbol sourceType, TypeSy
 
 
             // Preserve nullable modifiers as well.
-            // TODO: Do this only when feature is enabled?
-            var flagsBuilder = ArrayBuilder<bool>.GetInstance();
-            destinationType.AddNullableTransforms(flagsBuilder);
-            int position = 0;
-            int length = flagsBuilder.Count;
-            bool transformResult = resultType.ApplyNullableTransforms(flagsBuilder.ToImmutableAndFree(), ref position, out resultType);
-            Debug.Assert(transformResult && position == length);
-
-            Debug.Assert(resultType.Equals(sourceType, TypeCompareKind.IgnoreDynamicAndTupleNames)); // Same custom modifiers as source type.
-            Debug.Assert(resultType.Equals(destinationType,
-                                           TypeCompareKind.IgnoreCustomModifiersAndArraySizesAndLowerBounds | // Same object/dynamic and tuple names as destination type.
-                                           TypeCompareKind.CompareNullableModifiersForReferenceTypes)); // Same nullability as destination type.
+            // PROTOTYPE(NullableReferenceTypes): Set unknown nullability otherwise? 
```

Comment:
>// PROTOTYPE(NullableReferenceTypes): Set unknown nullability otherwise?  [](start = 12, length = 73)

I think this is not a question, we should do this now